Resources/Prototypes/DeltaV/Entities/Objects/Devices/pda.yml

```diff
@@ -38,6 +38,44 @@
     scanningEndSound:
       path: "/Audio/Items/Medical/healthscanner.ogg"
 
+- type: entity
+  parent: BasePDA
+  id: PharmacistPDA
+  name: pharmacist PDA
+  description: Blue, Orange, and odd stains... Has a built-in health analyzer.
+  components:
+  - type: Sprite
+    sprite: DeltaV/Objects/Devices/pda.rsi
+    layers:
+    - map: [ "enum.PdaVisualLayers.Base" ]
+    - state: "light_overlay"
+      map: [ "enum.PdaVisualLayers.Flashlight" ]
+      shader: "unshaded"
+      visible: false
+    - state: "id_overlay"
+      map: [ "enum.PdaVisualLayers.IdLight" ]
+      shader: "unshaded"
+      visible: false
+  - type: CartridgeLoader
+    diskSpace: 5
+    preinstalled:
+      - CrewManifestCartridge
+      - NotekeeperCartridge
+      - NewsReaderCartridge
+  - type: Pda
+    id: PharmacistIDCard
+    state: pda-pharmacist
+  - type: PdaBorderColor
+    borderColor: "#EA6400"
+    accentVColor: "#5D99BE"
+  - type: Icon
+    sprite: DeltaV/Objects/Devices/pda.rsi
+    state: pda-pharmacist
+  - type: HealthAnalyzer
```

Comment:
Psychologists do not get this, and the Pharmacist is even further removed from emergency medicine than Chemists. I would suggest omitting this or, alternatively, we decide that all Medical staff get health analyzers, period.

Reply:
Removed